src/styles/card.scss

```diff
@@ -0,0 +1,114 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+:root {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), CSS custom properties (variables) should be declared in a `:root` selector, but they should be defined in a separate file called `variables.scss` or similar. It seems that you have a `variables.scss` file, so make sure that all the variables are defined there and imported into this file.